app/assets/javascripts/events.js

```diff
@@ -25,9 +25,38 @@ var events = {
     }
 };
 
+var jitsi_events = {
```

Comment:
This is cool.  It turns out you can also make ajax calls just using the `remote: true` flag on a button:  https://blog.codeship.com/unobtrusive-javascript-via-ajax-rails/